### PR TITLE
Grid

### PR DESCRIPTION
**Pull request by lannisterrr**
Added Grid Component . 
2 items in a grid 
3 items in a grid.

https://user-images.githubusercontent.com/29068538/152767667-1ec2b992-ba1f-4f1c-9cbe-78fd58e16d17.mp4

### CODE REVIEW

**Review by ashishnnnnn — APPROVED** [review]
Code looks pretty much good :+1:

File: components/rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+/* Rating (e-commerce) */
+
+.read-only-wrapper {
+  flex: 1 1 10rem;
+}
+
+.interactive-wrapper {
+  flex: 1 1 10rem;
+  align-items: center;
+  gap: 0.3em;
```
Comment (hinanshisuthar) [inline components/rating/rating.css:12]: Use px for values <=10 px.

**Review by hinanshisuthar — COMMENTED** [review]
Everything looks good to me.

File: components/grid/grid.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
+@import url('../../style.css');
+
+/* Grid Component */
+
+.grid-component-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  box-shadow: -2px 3px 4px #888, -4px -3px 5px #888;
```
Comment (supminn) [inline components/grid/grid.css]: Add colors into variables.

File: components/grid/grid.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+.grid-component-container-2 {
```
Comment (supminn) [inline components/grid/grid.css]: rename it to `grid-container-with-area`

File: components/grid/grid.css [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 40px auto;
+  grid-template-areas:
+    'h h h'
+    'a c c';
```
Comment (supminn) [inline components/grid/grid.css]: not readable. try to use the full forms.

File: components/grid/grid.css [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  box-shadow: -2px 3px 4px #888, -4px -3px 5px #888;
+}
+
+.grid-item-1 {
```
Comment (supminn) [inline components/grid/grid.css]: could be named as `grid-header`

File: components/grid/grid.html [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+    />
+    <title>Grid</title>
+  </head>
+  <body>
+    <div class="component-border p-4">
+      <h3 class="heading-3 center-text t-c-3">Two Items in a Grid</h3>
+      <div class="grid-component-container">
+        <div class="f-6">
```
Comment (supminn) [inline components/grid/grid.html:18]: font-size could be shortened as `fs-6`

File: components/grid/grid.html [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Laborum, quo
+          exercitationem in officiis veniam nobis tempora commodi eveniet quam .
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Assumenda in
+          expedita aut ad unde maxime laboriosam earum. Unde rem accusamus ipsam
+          vero autem? Maxime quam ipsum, recusandae deserunt sit nemo?
+        </div>
+      </div>
+    </div>
+
+    <div class="component-border p-4 m-v-4">
```
Comment (supminn) [inline components/grid/grid.html:36]: `mx` and `my` for margins

File: components/rating/rating.css [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+    21% 91%,
+    32% 57%,
+    2% 35%,
+    39% 35%
+  );
+  border: 2px solid black;
+}
+
+.read-only-rating.empty {
+  background: var(--text-color-2);
```
Comment (supminn) [inline components/rating/rating.css:33]: you can specify the shade instead of number. 
Ex: light-grey

**Review by supminn — COMMENTED** [review]
Well structured code. You could improve on the naming conventions for utility classes.

**Pull request merged**